server/server.js

```diff
@@ -46,6 +49,12 @@ Server.prototype.terminate = function(done) {
 
 // Private API
 
+// 86400000:  number of milliseconds in 1 day
+var DEFAULT_TTL = 50000;
+
+// 900000:  number of milliseconds in 15 minutes
+var DEFAULT_PERIOD = 20000;
```

Comment:
These are dev/test values, which would be replaced by the values in the comments in production code.  

Some people do not like hardcoded default values, but they provide known fallbacks, and we expect them to be overridden by config or ENV settings.